pymc3/sampling.py

```diff
@@ -289,6 +296,9 @@ def sample(draws=500, step=None, init='auto', n_init=200000, start=None,
         Options for traceplot. Example: live_plot_kwargs={'varnames': ['x']}
     discard_tuned_samples : bool
         Whether to discard posterior samples of the tune interval.
+    compute_stats : bool, default=True
```

Comment:
Any ideas for a better name than `compute_stats`?

Reply:
`store_stats`?

Reply:
Hm. That might be confusing with the normal sampler stats.
Maybe `convergence_checks` or `convergence_stats` or the very long `compute_convergence_stats`?

Reply:
I think `compute_convergence_stats` suits.